File: solutions/python/leap/1/leap.py
def leap_year(year):
    # if year[-1] == 0 and year[-2] == 0:

    if (year % 4 == 0 and year % 100 != 0) or year % 400 == 0:
        
        # if year % 100 == 0 and year % 400 == 0:
            return True
    # elif year % 4 == 0:
        # return True
        
    
    # if (year % 100 == 0 and year % 400 == 0): 
    #     print('debug1')
    #     return True
    # elif year % 4 == 0 and year % 4 != 0:
    #     print('debug2')
    #     return True
    else:
        return False

File: solutions/c/leap/1/leap.c
#include "leap.h"
#include <stdio.h>

// bool leap_year(int year) {
//     printf("debug1 %i", year);
//     return false;
bool leap_year(int year)
{
    printf("%i", year);
    if((year % 4 == 0 && year % 100 != 0) || year % 400 == 0)
    {
        // printf("true");
        return true;
    }
    else 
        return false;
}
